underfs/pom.xml

```diff
@@ -39,7 +39,7 @@
       <dependency>
         <groupId>net.java.dev.jets3t</groupId>
         <artifactId>jets3t</artifactId>
-        <version>0.8.1</version>
+        <version>0.9.4</version>
```

Comment:
That sounds good. Are those changes in this PR (I only see changes to S3UnderFileSystem and not S3AUnderFileSystem)?